lib/_http_outgoing.js

```diff
@@ -482,7 +484,7 @@ OutgoingMessage.prototype.write = function(chunk, encoding, callback) {
 
 function writeAfterEndNT(self, err, callback) {
   self.emit('error', err);
-  if (callback) callback(err);
+  if (typeof callback === 'function') callback(err);
```

Comment:
This is an interesting one. Prior to the change a callback that isn't a function would throw. In this instance it would fail silently